### PR TITLE
feat: Add the Screenshot Labels management APIs

### PR DESCRIPTION
**Pull request by elliotbruneel**
This MR allows assign and unassigned label from screenshots

Fix: https://github.com/crowdin/crowdin-api-client-php/issues/132

### CODE REVIEW

File: src/CrowdinApiClient/Api/LabelApi.php [hunk 1]
```diff
@@ -88,4 +90,51 @@ public function delete(int $projectId, int $labelId)
         $path = sprintf('projects/%d/labels/%d', $projectId, $labelId);
         return $this->_delete($path);
     }
+
+    /**
+     * Assign Label to Screenshots
+     * @link https://developer.crowdin.com/api/v2/#operation/api.projects.labels.screenshots.post
```
Comment (andrii-bodnar) [inline src/CrowdinApiClient/Api/LabelApi.php:96]:
```suggestion
     * @link https://developer.crowdin.com/api/v2/#operation/api.projects.labels.screenshots.post API Documentation
```

File: src/CrowdinApiClient/Api/LabelApi.php [hunk 2]
```diff
@@ -88,4 +90,51 @@ public function delete(int $projectId, int $labelId)
...
+            'headers' => ['Content-Type' => 'application/json']
+        ];
+
+        return $this->client->apiRequest('post', $path, new ResponseModelListDecorator(Screenshot::class), $options);
+    }
+
+    /**
+     * Unassign Label from Screenshots
+     *
+     * @link https://developer.crowdin.com/api/v2/#operation/api.projects.labels.screenshots.deleteMany
```
Comment (andrii-bodnar) [inline src/CrowdinApiClient/Api/LabelApi.php:120]:
```suggestion
     * @link https://developer.crowdin.com/api/v2/#operation/api.projects.labels.screenshots.deleteMany API Documentation
```

File: src/CrowdinApiClient/Api/LabelApi.php [hunk 3]
```diff
@@ -88,4 +90,51 @@ public function delete(int $projectId, int $labelId)
...
+    public function assignScreenshots(int $projectId, int $labelId, array $data): ModelCollection
+    {
+        $path = sprintf('projects/%d/labels/%d/screenshots', $projectId, $labelId);
+
+        $options = [
+            'body' => json_encode($data),
+            'headers' => ['Content-Type' => 'application/json']
+        ];
+
+        return $this->client->apiRequest('post', $path, new ResponseModelListDecorator(Screenshot::class), $options);
```
Comment (andrii-bodnar) [inline src/CrowdinApiClient/Api/LabelApi.php:114]: Use the `_create` method

File: src/CrowdinApiClient/Api/LabelApi.php [hunk 4]
```diff
@@ -88,4 +90,51 @@ public function delete(int $projectId, int $labelId)
...
+    public function unassignScreenshots(int $projectId, int $labelId, array $data): ModelCollection
+    {
+        $path = sprintf('projects/%d/labels/%d/screenshots', $projectId, $labelId);
+
+        $options = [
+            'body' => json_encode($data),
+            'headers' => ['Content-Type' => 'application/json']
+        ];
+
+        return $this->client->apiRequest('delete', $path, new ResponseModelListDecorator(Screenshot::class), $options);
```
Comment (andrii-bodnar) [inline src/CrowdinApiClient/Api/LabelApi.php:138]: Use the `_delete` method

File: src/CrowdinApiClient/Api/LabelApi.php (hunk 3 above)
Comment (elliotbruneel) [reply src/CrowdinApiClient/Api/LabelApi.php:114]: As the method. `_create` return `ResponseModelDecorator` i'm unable the return the list with `ResponseModelListDecorator`

File: src/CrowdinApiClient/Api/LabelApi.php (hunk 4 above)
Comment (elliotbruneel) [reply src/CrowdinApiClient/Api/LabelApi.php:138]: As the method `_delete` does not used decorator i can't return the list of screenshots

File: src/CrowdinApiClient/Api/LabelApi.php (hunk 3 above)
Comment (andrii-bodnar) [reply src/CrowdinApiClient/Api/LabelApi.php:114]: Got you, didn't notice that, thank you

**Review by andrii-bodnar — APPROVED** [review]
@elliotbruneel thank you!

**Pull request merged**